cms/djangoapps/contentstore/management/commands/tests/test_cleanup_assets.py

```diff
@@ -23,7 +24,8 @@ class ExportAllCourses(ModuleStoreTestCase):
     def setUp(self):
         """ Common setup. """
         self.content_store = contentstore()
-        self.module_store = modulestore()
+        # pylint: disable=protected-access
+        self.module_store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.mongo)
```

Comment:
The way that assets are accessed (the usage of _id) seems specific to old mongo and fails with split.